dask/dataframe/core.py

```diff
@@ -2394,13 +2436,19 @@ def info(self, buf=None, verbose=False, memory_usage=False):
 for name in ['add', 'sub', 'mul', 'div',
              'truediv', 'floordiv', 'mod', 'pow',
              'radd', 'rsub', 'rmul', 'rdiv',
-             'rtruediv', 'rfloordiv', 'rmod', 'rpow']:
+             'rtruediv', 'rfloordiv', 'rmod', 'rpow', 'eq']:
```

Comment:
Relic of my personal testing of the function that got left in. I backed it back out